src/gc/gcee.cpp

```diff
@@ -392,7 +392,7 @@ size_t GCHeap::GetNow()
 
 void ProfScanRootsHelper(Object** ppObject, ScanContext *pSC, DWORD dwFlags)
 {
-#if defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
+#if defined(GC_PROFILING) && defined(FEATURE_EVENT_TRACE) 
```

Comment:
Is this change really correct? I can see the original form of the condition used at about 50 places in CoreCLR sources.